Easydict/App/Localizable.xcstrings

```diff
@@ -1140,13 +1140,13 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Force get selected text when auto get selected text failed (Experimental)"
+            "value" : "Allow to use Cmd + C to force to get selected text when regular method fails (experimental feature)"
```

Comment:
Could you make this sentence shorter? And I think current sentence is not particularly clear to users what issue it will fix.

Reply:
ok, changed to `Allow to use force to get selected text (experimental feature)`.